config/rapidez/frontend.php

```diff
@@ -33,11 +33,14 @@
         // Attach additional indexes to the autocomplete
         // Uses the views in rapidez::layouts.partials.header.autocomplete
         'additionals' => [
-            'categories' => ['name^3', 'description'],
+            'categories' => [
+                'fields' => ['name^3', 'description'],
+                'size' => 3,
+            ],
```

Comment:
Isn't this a breaking change? Can we make this non-breaking and backwards compatible somehow?

Reply:
@Jade-GG This shouldn't be a breaking change right?

Reply:
This uses functionality that's been in the core ever since this feature was added, so there should not be any backwards compatibility issues.

Reply:
When you upgrade the core, forget to copy over the config changes you'll end up with unexpected behavior I think? Or the other way around, you haven't published the config but did overwrite the template. We can also make this a v3 feature?

Reply:
I will revert the changes for now so you can override the config if it is needed to change the results in the autocomplete.

Reply:
If you don't update the config you will get 10 products and 10 categories shown (or whatever else you have configured), indeed. Would that be considered breaking?

(Also, the size for categories should not need to be specified here, as the standard size is set to 3 below)

Reply:
I didn't ask for a revert? I'm just discussing the options and possible issues here.

Mm, but `additionals.categories` changes to `additionals.categories.fields`, that's not breaking?

Reply:
Nope, both options have actually been supported since the addition of this config section 😇

Reply:
Wow, next time just explain that directly and points me to the code please: https://github.com/rapidez/core/blob/93800693c106130bf6721401948e777fa6d11a62/resources/js/components/Search/Autocomplete.vue#L56-L57, so the revert can be reverted?